PubNub/Data/PNConfiguration.m

```diff
@@ -123,7 +123,7 @@ - (void)setPresenceHeartbeatValue:(NSInteger)presenceHeartbeatValue {
     _presenceHeartbeatValue = presenceHeartbeatValue;
     if (self.presenceHeartbeatInterval == 0) { 
         
-        _presenceHeartbeatInterval = (NSInteger)(_presenceHeartbeatValue * 0.5f);
+        _presenceHeartbeatInterval = (NSInteger)(_presenceHeartbeatValue * 0.5f) - 1;
```

Comment:
Should we document that this is -1? Why is that exactly?

Reply:
It looks like all other SDK used this formula. I just added what I forgot (for pretty long time).
Half of heartbeat value - 1 second.